### ISSUE TITLE
some error in inference 

### ISSUE THREAD

**Issue by ChengHSUHSU** [hpcaitech/Open-Sora#208]
It is in scripts/inference.py.

I execute the python code, then the error is in 93-line, but there is no any error message to response.

<img width="544" alt="Screenshot 2024-03-24 at 9 28 00 PM" src="https://github.com/hpcaitech/Open-Sora/assets/76766937/4d8e5b1c-6485-4f47-8b3c-1f943a674f48">

thanks,

**Comment by pokameng** [comment]
me too

**Comment by zhengzangw** [comment]
Could you please provide more information? Such as how the error appears in terminal and what's your environment, cuda version and GPU? @JThh will help yo to solve it.

**Comment by pokameng** [comment]
ok
My environment is:
`absl-py                   2.1.0
accelerate                0.28.0
addict                    2.4.0
aiosignal                 1.3.1
annotated-types           0.6.0
apex                      0.1
appdirs                   1.4.4
attrs                     23.2.0
bcrypt                    4.1.2
beautifulsoup4            4.12.3
certifi                   2024.2.2
cffi                      1.16.0
cfgv                      3.4.0
charset-normalizer        3.3.2
click                     8.1.7
colossalai                0.3.6
contexttimer              0.3.3
contourpy                 1.2.0
cryptography              42.0.5
cycler                    0.12.1
decorator                 5.1.1
Deprecated                1.2.14
diffusers                 0.27.2
distlib                   0.3.8
docker-pycreds            0.4.0
einops                    0.7.0
fabric                    3.2.2
filelock                  3.13.1
flash-attn                2.5.6
fonttools                 4.50.0
frozenlist                1.4.1
fsspec                    2024.3.1
ftfy                      6.2.0
gdown                     5.1.0
gitdb                     4.0.11
GitPython                 3.1.42
google                    3.0.0
grpcio                    1.62.1
huggingface-hub           0.21.4
identify                  2.5.35
idna                      3.6
importlib_metadata        7.1.0
invoke                    2.2.0
Jinja2                    3.1.3
jsonschema                4.21.1
jsonschema-specifications 2023.12.1
kiwisolver                1.4.5
Markdown                  3.6
markdown-it-py            3.0.0
MarkupSafe                2.1.5
matplotlib                3.8.3
mdurl                     0.1.2
mmengine                  0.10.3
mpmath                    1.3.0
msgpack                   1.0.8
networkx                  3.2.1
ninja                     1.11.1.1
nodeenv                   1.8.0
numpy                     1.26.4
nvidia-cublas-cu12        12.1.3.1
nvidia-cuda-cupti-cu12    12.1.105
nvidia-cuda-nvrtc-cu12    12.1.105
nvidia-cuda-runtime-cu12  12.1.105
nvidia-cudnn-cu12         8.9.2.26
nvidia-cufft-cu12         11.0.2.54
nvidia-curand-cu12        10.3.2.106
nvidia-cusolver-cu12      11.4.5.107
nvidia-cusparse-cu12      12.1.0.106
nvidia-nccl-cu12          2.19.3
nvidia-nvjitlink-cu12     12.4.99
nvidia-nvtx-cu12          12.1.105
opencv-python             4.9.0.80
opensora                  1.0.0
packaging                 24.0
paramiko                  3.4.0
pillow                    10.2.0
pip                       23.3.1
platformdirs              4.2.0
pre-commit                3.7.0
protobuf                  4.25.3
psutil                    5.9.8
pyav                      12.0.5
pycparser                 2.21
pydantic                  2.6.4
pydantic_core             2.16.3
Pygments                  2.17.2
PyNaCl                    1.5.0
pyparsing                 3.1.2
PySocks                   1.7.1
python-dateutil           2.9.0.post0
PyYAML                    6.0.1
ray                       2.10.0
referencing               0.34.0
regex                     2023.12.25
requests                  2.31.0
rich                      13.7.1
rpds-py                   0.18.0
safetensors               0.4.2
sentencepiece             0.2.0
sentry-sdk                1.43.0
setproctitle              1.3.3
setuptools                68.2.2
six                       1.16.0
smmap                     5.0.1
soupsieve                 2.5
sympy                     1.12
tensorboard               2.16.2
tensorboard-data-server   0.7.2
termcolor                 2.4.0
timm                      0.9.16
tokenizers                0.15.2
tomli                     2.0.1
torch                     2.2.1
torchaudio                2.2.1
torchvision               0.17.1
tqdm                      4.66.2
transformers              4.39.1
triton                    2.2.0
typing_extensions         4.10.0
urllib3                   2.2.1
virtualenv                20.25.1
wandb                     0.16.4
wcwidth                   0.2.13
Werkzeug                  3.0.1
wheel                     0.41.2
wrapt                     1.16.0
xformers                  0.0.25
yapf                      0.40.2
zipp                      3.18.1`

when the code runs the line 101:

`        samples = vae.decode(samples.to(dtype))
`
the program is stopped

My gpu is A6000
@zhengzangw

**Comment by pokameng** [comment]
![image](https://github.com/hpcaitech/Open-Sora/assets/47957015/a5da3d54-bd94-4376-9855-2eae922380aa)
when i run the inference with the config of 16*512*512,something is wrong

**Comment by JThh** [comment]
Can you run `pip list | grep torch`? I see you may not have the correct `torch` installed to facilitate GPU use. Check if `nvcc --version` and `python -c "import torch; print(torch.__version__); print(torch.version.cuda);"` is consistent.

**Comment by pokameng** [comment]
![image](https://github.com/hpcaitech/Open-Sora/assets/47957015/49ffdd46-135b-4940-a578-dd1cee7911a0)
@JThh

**Comment by JThh** [comment]
Looks good to me. What was your command to run `inference.py`?

**Comment by whk6688** [comment]
the same to me:
![image](https://github.com/hpcaitech/Open-Sora/assets/17873056/3658ead6-cc82-402d-91c5-73f0bd9965f1)


Missing key(s) instate_dict:"['pos_embed','pos_embed_temporal']","['pos_embed','pos_embed_temporal']"

my command: 
torchrun --standalone --nproc_per_node 1 scripts/inference.py configs/opensora/inference/16x256x256.py --ckpt-path /home/wanghaikuan/.cache/huggingface/hub/models--DeepFloyd--t5-v1_1-xxl/snapshots/ --prompt-path ./assets/texts/t2v_samples.txt

i only has one gpu card.

**Comment by ChengHSUHSU** [comment]
@ALL
thank [zhengzangw](https://github.com/zhengzangw),
I use the Dockerfile and build dev. env
reference : (https://github.com/hpcaitech/Open-Sora/pull/159/files)

It is sucessfully run inference 

Moreover, cuda version is 11.8
